ui/src/Components/Learn.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { Table } from 'reactstrap';
+import { Link } from 'react-router-dom';
+
+export default class ProfilePage extends React.Component {
+    constructor(props){
+      super(props);
+    }
+  
+    render(){
+      return(
+        <Table className="row-buffer" responsive>
+            <h3> Start Learning Here! </h3>
```

Comment:
I think you need to add some customized CSS. All other pages are more cohesive in terms of the titles and text. 
![image](https://user-images.githubusercontent.com/43486058/101220163-338d0f00-3653-11eb-80d6-717e3062b7e3.png)
![image](https://user-images.githubusercontent.com/43486058/101220717-29b7db80-3654-11eb-91c6-4c744ed844d9.png)

^ This shows the titles and text on other pages and how they follow the same styling vs the below shows the titles/text in the LMS

![image](https://user-images.githubusercontent.com/43486058/101220785-4ce28b00-3654-11eb-9dc3-3b129b0417cb.png)
![image](https://user-images.githubusercontent.com/43486058/101220791-4fdd7b80-3654-11eb-8655-9a8d0915599a.png)